src/extensions/mvs/tree/mvs/mvs-representations.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Please rename this file to "mvs-tree-representations.ts", to keep consistency with "mvs-tree-primitives.ts"